src/Components/Sidebar/Sidebar.jsx

```diff
@@ -1,18 +1,36 @@
 import React, { useState } from 'react';
-import { Link } from 'react-router-dom';
+import { Link, useNavigate } from 'react-router-dom';
 import './Sidebar.css';
 import { ICONS, ACTIVE_ICONS, ACTIVE_CLASS } from '../../constants';
+import { signOut } from 'firebase/auth';
+import { auth } from '../../config/firebase.config';
+import { toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+import { TOAST_MESSAGES } from '../../constants';
 const Sidebar = () => {
+
+  const navigate = useNavigate();
+  const handleLogout = async () => {
+    try {
+      await signOut(auth);
+      toast.success(TOAST_MESSAGES.SIGN_OUT_SUCCESS);
+      navigate("/login");
```

Comment:
move this route to a constants route file

Reply:
Done